Console/Command/ReplicaSyncCommand.php

```diff
@@ -27,12 +27,12 @@ public function __construct(
         protected ReplicaManagerInterface $replicaManager,
         protected ProductHelper           $productHelper,
         protected StoreManagerInterface   $storeManager,
-        State                             $state,
+        AppState                          $appState,
```

Comment:
This was largely a cosmetic change - but while experimenting with ReplicaState in the class I realized that it could get confused with AppState as they are two different things. I left the alias in case I decided to use both states in the future. If you think it's best to leave as it was originally I'll remove the change.